src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -149,9 +149,13 @@ ldap but it felt unnatural.
     roles: Set[ResourceRoleName],
     actions: Set[ResourceAction],
     public: Boolean)
-    extends WorkbenchGroup
+    extends WorkbenchGroup {
+  def metadata: AccessPolicyMetadata = AccessPolicyMetadata(id, email, roles, actions, public)
```

Comment:
Is it time to update the comment on AccessPolicy? We're not constrained by ldap anymore. Or is it useful to still say that's where it comes from?